benchmarks/gaussian_localisation/gaussian_test.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import dask
+import glob
+import re
+import sys
+import os
+
+import deepblink as pink
+import numpy as np
+import scipy.optimize as opt
+
+sys.path.append("../")
+from util import delayed_results
+
+EPS = 1e-4
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--datasets")
+    parser.add_argument("--models")
+    args = parser.parse_args()
+    return args
+
+
+@dask.delayed
+def delayed_normalize_pink(image):
+    return pink.data.normalize_image(image)
+
+
+def gauss_2d(xy, amplitude, x0, y0, sigma_xy, offset):
+    """2D gaussian."""
+    x, y = xy
+    x0 = float(x0)
+    y0 = float(y0)
+    gauss = offset + amplitude * np.exp(
+        -(
+            ((x - x0) ** (2) / (2 * sigma_xy ** (2)))
+            + ((y - y0) ** (2) / (2 * sigma_xy ** (2)))
+        )
+    )
+    return gauss
+
+
+def gauss_single_spot(
+    image: np.ndarray, c_coord: float, r_coord: float, crop_size: int
+):
+    """Gaussian prediction on a single crop centred on spot."""
+    start_dim1 = np.max([int(np.round(r_coord - crop_size // 2)), 0])
+    if start_dim1 < len(image) - crop_size:
+        end_dim1 = start_dim1 + crop_size
+    else:
+        start_dim1 = len(image) - crop_size
+        end_dim1 = len(image)
+
+    start_dim2 = np.max([int(np.round(c_coord - crop_size // 2)), 0])
+    if start_dim2 < len(image) - crop_size:
+        end_dim2 = start_dim2 + crop_size
+    else:
+        start_dim2 = len(image) - crop_size
+        end_dim2 = len(image)
+
+    crop = image[start_dim1:end_dim1, start_dim2:end_dim2]
+
+    x = np.arange(0, crop.shape[1], 1)
+    y = np.arange(0, crop.shape[0], 1)
+    xx, yy = np.meshgrid(x, y)
+
+    # Guess intial parameters
+    x0 = int(crop.shape[0] // 2)  # Middle of the crop
+    y0 = int(crop.shape[1] // 2)  # Middle of the crop
+    sigma = max(*crop.shape) * 0.1  # 10% of the crop
+    amplitude_max = np.max(crop) / 2  # Maximum value of the crop
+    initial_guess = [amplitude_max, x0, y0, sigma, 0]
+
+    lower = [np.min(crop), 0, 0, 0, np.min(crop)]
+    upper = [
+        np.max(crop) + EPS,
+        crop.shape[0],
+        crop.shape[1],
+        np.inf,
+        np.max(crop) + EPS,
+    ]
+    bounds = [lower, upper]
+
+    try:
+        popt, _ = opt.curve_fit(
+            gauss_2d,
+            (xx.ravel(), yy.ravel()),
+            crop.ravel(),
+            p0=initial_guess,
+            bounds=bounds,
+        )
+    except:
+        return r_coord, c_coord
+
+    x0 = popt[1] + start_dim2
+    y0 = popt[2] + start_dim1
+
+    # if predicted spot is out of the border of the image
+    if x0 >= image.shape[1] or y0 >= image.shape[0]:
+        return r_coord, c_coord
+
+    return y0, x0
+
+
+@dask.delayed
+def gauss_single_image(image: np.ndarray, mask: np.ndarray, crop_size: int = 4):
+    """Gaussian prediction on a single image."""
+    coord_list = pink.data.get_coordinate_list(mask.squeeze(), 512)
```

Comment:
Here you take from the mask -> ground truth, not the output from deepblink or how is mask defined?